src/common/runtime-bind.js

```diff
@@ -0,0 +1,23 @@
+define([
+	"./runtime-key"
+], function( runtimeKey ) {
+
+return function( args, cldr, fn, runtimeArgs ) {
+	var fnName = /^function\s+([\w\$]+)\s*\(/.exec( fn.toString() )[ 1 ],
+		locale = cldr.locale;
+
+	fn.runtimeKey = runtimeKey( fnName, locale, args );
```

Comment:
Actually, it's passed twice by the normal modules. But, it's passed once by the runtime modules (e.g., https://github.com/jquery/globalize/blob/2baa3317e2726cc6d5f9c0f8895958f78ab9aa34/src/date-runtime.js#L28).

I'm considering to handle both worlds with:
```diff
diff --git a/src/common/runtime-key.js b/src/common/runtime-key.js
index 45b64e5..4d95f06 100644
--- a/src/common/runtime-key.js
+++ b/src/common/runtime-key.js
@@ -3,8 +3,9 @@ define([
 ], function( stringHash ) {
 
 return function( fnName, locale, args ) {
-       var hash;
-       hash = stringHash( fnName + locale + JSON.stringify( args ) );
+       var argsStr, hash;
+       argsStr = typeof args === "string" ? args : JSON.stringify( args );
+       hash = stringHash( fnName + locale + argsStr );
        return hash > 0 ? "a" + hash : "b" + Math.abs( hash );
 };
```